src/main/java/hello/controller/IssueController.java

```diff
@@ -0,0 +1,181 @@
+            OkHttpClient client = new OkHttpClient();
+
+            Request request = new Request.Builder().url(url).build();
+            Response response = client.newCall(request).execute();
+
+            return new Gson().fromJson(response.body().string(), CodeGenGetResponse.class);
+        } catch (IOException e) {
+            e.printStackTrace();
+            return new CodeGenGetResponse("Failure", "");
+        }
+    }
+
+    private void fixIssue1(Integer methodId, String modifiers) {
+        String [] modifierList = {"public", "protected", "private", "static", "abstract", "synchronized", "transient", "volatile", "final", "native", "strictfp"};
+        String newModifiers = "";
+        for(String modifier : modifierList) {
+            if (modifiers.contains(modifier)) {
+                newModifiers += modifier + ", ";
+            }
+        }
+        if (newModifiers.endsWith(", ")) {
+            newModifiers = newModifiers.substring(0, newModifiers.length() - 2);
+        }
+        newModifiers = "[" + newModifiers + "]";
+        Session session = driver.session();
+        session.run("match (n) where id(n)=" + methodId + " set n.modifiers=\"" + newModifiers + "\"");
+    }
+
+    private void fixIssue3(Integer methodId, String modifiers) {
+        String [] modifierList = {"public", "protected", "private", "static", "abstract", "synchronized", "transient", "volatile", "final", "native", "strictfp"};
+        String newModifiers = "";
+        for(String modifier : modifierList) {
+            if (modifiers.contains(modifier) || modifier.equals("synchronized")) {
+                newModifiers += modifier + ", ";
+            }
+        }
+        if (newModifiers.endsWith(", ")) {
+            newModifiers = newModifiers.substring(0, newModifiers.length() - 2);
+        }
+        newModifiers = "[" + newModifiers + "]";
+        Session session = driver.session();
+        session.run("match (n) where id(n)=" + methodId + " set n.modifiers=\"" + newModifiers + "\"");
+    }
+
+    @GetMapping(path="/fix", produces = "application/json")
+    public String fixIssue(@RequestParam(value="issueId") Integer issueId, @RequestParam(value="sandBoxUrl") String sandBoxUrl) {
+        setNeo4jDriver(sandBoxUrl);
+        CodeGenPostRequest codeGenPostRequest = new CodeGenPostRequest(sandBoxUrl, Constants.USER, Constants.PASS);
+        Session session = driver.session();
+        List<Record> result;
+        switch (issueId) {
+            case 1:
+                result = session.run(Constants.QUERY1).list();
+                for(Record record : result) {
+                    codeGenPostRequest.fileIds.add(record.get("file_id").asInt());
+                    fixIssue1(record.get("method_id").asInt(), record.get("modifiers").asString());
+                }
+
+                Integer fixId = sendPostRequestCodeGen(Constants.CODEGEN_URL, new Gson().toJson(codeGenPostRequest)).id;
+
+                if (fixId == -1) {
```

Comment:
You're effectively treating the -1 as a flag that indicates an exception. Use exceptions instead. Create a relevant checked exception and throw it.